aws/data_source_aws_route53_zone_test.go

```diff
@@ -121,5 +162,15 @@ data "aws_route53_zone" "by_zone_id" {
 data "aws_route53_zone" "by_name" {
   name = "${data.aws_route53_zone.by_zone_id.name}"
 }
-`, rInt, rInt, rInt, rInt)
+
+resource "aws_service_discovery_private_dns_namespace" "test_service_discovery" {
+  name        = "test.acc-sd-%d"
+  vpc         = "${aws_vpc.test.id}"
+}
+
+data "aws_route53_zone" "service_discovery_by_vpc" {
+  name   = "${aws_service_discovery_private_dns_namespace.test_service_discovery.name}"
+  vpc_id = "${aws_vpc.test.id}"
+}
+`, rInt, rInt, rInt, rInt, rInt)
```

Comment:
Instead of repeatedly defining the same `fmt.Sprintf()` arguments, the format verbs support a 1-based index argument (e.g. `%[1]d` instead of `%d`) and this list can be simplified to a single `rInt` argument.

After replacing all `%d` above with `%[1]d`:

```suggestion
`, rInt)
```